masterlist.yaml

```diff
@@ -12951,6 +12951,58 @@ plugins:
         util: '[SSEEdit v4.0.3](https://www.nexusmods.com/skyrimspecialedition/mods/164)'
         itm: 2
 
+  - name: 'fortdawnguardimmersive.esp'
+    url:
+      - link: 'https://www.nexusmods.com/skyrimspecialedition/mods/40436/'
+        name: 'Immersive Fort Dawnguard'
+    msg:
+      - <<: *patchIncluded
+        subs: [ '[Enhanced Lights and FX](https://www.nexusmods.com/skyrimspecialedition/mods/2424)' ]
+        condition: 'active("EnhancedLightsandFX.esp") and not active("FDIELnFXPatch.esp")'
+      - <<: *patchIncluded
+        subs: [ '[Medieval Lanterns of Skyrim](https://www.nexusmods.com/skyrimspecialedition/mods/27622)' ]
+        condition: 'active("Medieval Lanterns of Skyrim.esp") and not active("FDIMLOSPatch.esp")'
+      - <<: *patchIncluded
+        subs: [ '[RLO Interiors](https://www.nexusmods.com/skyrimspecialedition/mods/844)' ]
+        condition: 'active("RLO - Interiors.esp") and not active("FDIRLOInteriorsPatch.esp")'
+      - <<: *patchIncluded
+        subs: [ '[Skyrim Project Optimization SE](https://www.nexusmods.com/skyrimspecialedition/mods/14084/)' ]
+        condition: 'active("Skyrim Project Optimization.*\.es(p|m)") and not active("FDISPOPatch.esp")'
```

Comment:
The included patch only works for `Skyrim Project Optimization - Full Version.esm`.

![image](https://user-images.githubusercontent.com/1944639/93152949-ece4e400-f6f7-11ea-9d8a-07920bf1e8ec.png)

```suggestion
        condition: 'active("Skyrim Project Optimization - Full Version.esm") and not active("FDISPOPatch.esp")'
```

Reply:
Yeah, I just did that since the incompatibilities will still be present with the other versions of SPO. I thought it'd be best if users see the message, get the patch, see they need the ESM version, and switch to it. I guess it's best if the author addresses the issue instead.